### ISSUE TITLE
Fully migrate to AndroidX to disable Jetifier 

### ISSUE THREAD

**Issue by androideveloper** [Instabug/Instabug-React-Native#695]
Hey,

We are migrating away from support libraries to disable Jetifier and noticed that one of the dependencies that still has android support libs is coming from Instabug-react-native sdk. Could you please have a look? 🙏 

```
The following libraries used by project ':instabug-reactnative' depend on legacy support libraries. To disable Jetifier, you will need to use AndroidX-supported versions of these libraries.
        com.android.support:multidex:1.0.3 (Project ':instabug-reactnative', configuration 'debugAndroidTestCompileClasspath' -> com.android.support:multidex:1.0.3)
```

support.multidex should be replaced with androidx multidex https://developer.android.com/jetpack/androidx/releases/multidex
There are 2 places that should be fixed https://github.com/Instabug/Instabug-React-Native/search?q=com.android.support%3Amultidex

**Comment by DavidMina96** [comment]
Thank you for your patience @androideveloper. We've created a snapshot for you with the AndroidX version of Multidex library. You can install it using this command:
`npm install https://github.com/Instabug/Instabug-React-Native#snapshot/10.13.0-MOB-8672`

Please let us know if you have any feedback :)

**Comment by androideveloper** [comment]
Hi @DavidMina96 
I checked the commit https://github.com/Instabug/Instabug-React-Native/commit/9098b7a6db1be465753a75afe00f233f1d3c0e0b and it looks good, that was the only location having support libs. When are you planning to have a new release?

**Comment by DavidMina96** [comment]
Sounds great @androideveloper! The ETA for the next release is 4 weeks.

**Comment by androideveloper** [comment]
Sounds good! Please keep me posted here once you release 🙏

**Comment by DavidMina96** [comment]
@androideveloper The fix is released in v11.0.0. 🚀
